pom.xml

```diff
@@ -104,6 +104,16 @@
 			<artifactId>javaparser-core</artifactId>
 			<version>3.25.9</version>
 		</dependency>
+		<dependency>
+			<groupId>com.github.javaparser</groupId>
+			<artifactId>javaparser-symbol-solver-core</artifactId>
+			<version>3.25.9</version>
```

Comment:
Please use a property to enusre that both javaparser dependencies use the same version.